zebra-network/src/protocol/external/inv.rs

```diff
@@ -63,39 +77,43 @@ impl From<block::Hash> for InventoryHash {
     }
 }
 
+impl From<transaction::WtxId> for InventoryHash {
+    fn from(wtx_id: transaction::WtxId) -> InventoryHash {
+        InventoryHash::Wtx(wtx_id)
+    }
+}
+
 impl ZcashSerialize for InventoryHash {
     fn zcash_serialize<W: Write>(&self, mut writer: W) -> Result<(), std::io::Error> {
-        let (code, bytes): (_, &[u8]) = match self {
-            InventoryHash::Error => (0, &[0; 32]),
-            InventoryHash::Tx(hash) => (1, &hash.0),
-            InventoryHash::Block(hash) => (2, &hash.0),
-            InventoryHash::FilteredBlock(hash) => (3, &hash.0),
-            InventoryHash::Wtx(bytes) => (5, bytes),
```

Comment:
Once we stop storing `WtxId`s as a byte array, it gets much harder to return a slice here.
(We'd have to create an extra binding to store the temporary clone.)